src/main/resources/catalogclient/application.properties

```diff
@@ -0,0 +1 @@
+pa.calendar.catalog-url=http://localhost:8080/catalog
```

Comment:
pa.calendar.catalog-url I think we usually use the pattern: pa.calendar.catalog.url